README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
```

Comment:
Not only just to rotate shapes but to bring it down more quickly.
Do not hold it (the down arrow key) for more time as the next block gets down quickly 
This might not allow you to rotate the shape.